### ISSUE TITLE
"Downloaded empty artifact" when `oras pull` from docker.io

### ISSUE THREAD

**Issue by codewithcheese** [oras-project/oras#453]
Hey all, I was expecting to be able to pull a docker hub image using ORAS CLI.

`oras pull docker.io/library/ubuntu:latest`

results in:

```
Downloaded empty artifact
Pulled docker.io/library/ubuntu:latest
Digest: sha256:b6b83d3c331794420340093eb706a6f152d9c1fa51b262d9bf34594887c2c7ac
```

With no image on disk, I would expect it to write a tar like any other mime type.

I am writing a TypeScript client to push/pull images AND artifacts and would prefer to only have a single implementation using oras for all the mime types. Don't necessarily need docker hub support, simply using it for testing an image pull.

**Comment by qweeah** [comment]
ORAS v0.13.0 cannot be used to download an image. It will only download layers annotated with [OCI image title](https://github.com/opencontainers/image-spec/blob/main/specs-go/v1/annotations.go#L52)

**Comment by shizhMSFT** [comment]
As @qweeah has mentioned, the current oras CLI can only pull named artifacts.

In the future version of `oras`, we will support `oras copy` from a remote registry to a local OCI store or a tarball in OCI layout.

**Comment by FeynmanZhou** [comment]
> As @qweeah has mentioned, the current oras CLI can only pull named artifacts.
> 
> In the future version of `oras`, we will support `oras copy` from a remote registry to a local OCI store or a tarball in OCI layout.

Is it corresponding to this [feature request](https://github.com/oras-project/oras/issues/378)?

**Comment by shizhMSFT** [comment]
@FeynmanZhou Yes, exactly.

**Comment by qweeah** [comment]
Closing as it's already supportted in `oras cp`.

**Comment by qweeah** [comment]
@codewithcheese You may build from main branch and try below commands to download a image to the `downloaded` folder in current working directory
```
oras cp docker.io/library/ubuntu:latest --to-oci-layout downloaded
```

This feature will be included in oras 1.0.0-rc.1 release planned for end of Jan 2023.
